tests/check/test_helpers.c

```diff
@@ -141,8 +149,71 @@ pull_seis(signed_video_t *sv, test_stream_item_t **item)
   return pulled_seis;
 }
 
+static bool
+read_file_content(const char *filename, char **content, size_t *content_size)
+{
+  bool success = false;
+  FILE *fp = NULL;
+  char full_path[MAX_PATH_LENGTH] = {0};
+  char cwd[MAX_PATH_LENGTH] = {0};
+
+  *content = NULL;
+  *content_size = 0;
+
+  if (!getcwd(cwd, sizeof(cwd))) {
+    goto done;
+  }
+
+  // Find the root location of the library.
+  char *lib_root = NULL;
+  char *next_lib_root = strstr(cwd, "signed-video-framework");
+  while (next_lib_root) {
+    lib_root = next_lib_root;
+    next_lib_root = strstr(next_lib_root + 1, "signed-video-framework");
+  }
+  if (!lib_root) {
+    goto done;
+  }
+  // Terminate string after lib root.
+  memset(lib_root + strlen("signed-video-framework"), '\0', 1);
+
+  // Get certificate chain from folder tests/.
+  strcat(full_path, cwd);
+  strcat(full_path, "/tests/");
+  strcat(full_path, filename);
+
+  fp = fopen(full_path, "rb");
+  if (!fp) {
+    goto done;
+  }
+
+  fseek(fp, 0L, SEEK_END);
+  size_t file_size = ftell(fp);
```

Comment:
I think we should check if file_size is zero(empty file) and return NOT success at this point.

Nothing will crash in the code if read file is empty but it probably good to fail here instead of forwarding the error to later.